### DISCUSSION TITLE
[feature request] chat like support

### DISCUSSION THREAD

**Discussion by solsticedhiver** [jcorporation/myMPD#1229] [Ideas]
hi.
I am used to use either IRC, matrix, discord, or even weird real time chat system, to ask question on a dedicated channel/room.

Is there one for mympd? Can you create one?
There is only the need for a handfull of motivated people to moderate it, if need be, and this could live this life without much intervention apart knowledgeable user contribution an answers...

**Comment by jcorporation** [comment]
Creating is no problem, but who moderates it? myMPD is an one man project.

There is not much activity here in the discussion, I doubt an other channel will be almost all the time empty.
